ubuntu/common/hpc-tuning.sh

```diff
@@ -29,20 +29,22 @@ sysctl -p
 
 # Install WALinuxAgent
 apt-get install python3-setuptools
-git clone https://github.com/Azure/WALinuxAgent.git
-pushd WALinuxAgent/
-git fetch origin pull/2308/head:version-2.4.0.1
-git checkout version-2.4.0.1
+WAAGENT_VERSION=v2.5.0.1
+$COMMON_DIR/write_component_version.sh "WAAGENT" ${WAAGENT_VERSION}
+DOWNLOAD_URL=https://github.com/Azure/WALinuxAgent/archive/refs/tags/pre-${WAAGENT_VERSION}.tar.gz
+wget ${DOWNLOAD_URL}
+tar -xvf $(basename ${DOWNLOAD_URL})
+pushd WALinuxAgent-pre-${WAAGENT_VERSION}/
 python3 setup.py install --register-service
 popd
 
 # Configure WALinuxAgent
 sed -i -e 's/# OS.EnableRDMA=y/OS.EnableRDMA=y/g' /etc/waagent.conf
-echo "Extensions.GoalStatePeriod=120" | sudo tee -a /etc/waagent.conf
+echo "Extensions.GoalStatePeriod=300" | sudo tee -a /etc/waagent.conf
```

Comment:
Dont we need this for CentOS also?

Reply:
@mvrequa  - please review the waagent config changes.

Reply:
> Dont we need this for CentOS also?

Added

Reply:
May we add this? If not set then GoalStatePeriod is used for the initial converge.
 Extensions.InitialGoalStatePeriod=6